.github/CODEOWNERS

```diff
@@ -9,6 +9,8 @@
 # Rules are matched bottom-to-top, so one team can own subdirectories
 # and another the rest of the directory.
 
+/                                       @DataDog/agent-platform
+
```

Comment:
```suggestion

/rtloader/                              @DataDog/agent-core
```

Reply:
(good catch, I'll add this a bit below to preserve alphabetical ordering though)

Reply:
done